src/utils.js

```diff
@@ -244,6 +244,36 @@ export const getNestedProperty = (path, object) => {
   });
 };
 
+/**
+ * @function getNestedPropertyWithFallback
+ *
+ * @description
+ * Parse the path passed and get the nested property at that path, or return the
+ * given noMatchValue if nothing is found at that path.
+ *
+ * @param {*} noMatchValue the fallback value
+ * @param {Array<number|string>|number|string} path the path to retrieve values from the object
+ * @param {*} object the object to get values from
+ * @returns {*} the retrieved values
+ */
+export const getNestedPropertyWithFallback = (noMatchValue, path, object) => {
```

Comment:
u shouldn't need to create this new function, just augment the existing `getNestedProperty`:

```javascript
export const getNestedProperty = (path, object, noMatchValue) => {
  ...
  return onMatchAtPath(
    parsedPath,
    object,
    (ref, key) => {
      return !!ref && ref[key] !== void 0
    },
    false,
    noMatchValue
  );
};
```

`noMatchValue` is always returned when nothing is passed, and will default to `undefined`, so no need for another function to do the same thing.